aiohttp/helpers.py

```diff
@@ -514,3 +515,22 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
     def _cancel_task(self):
         self._cancelled = self._task.cancel()
+
+
+def stop_listening(srv, loop):
+    """Stop listening Server instance.
+
+    Close all listening sockets leaving accepted connections untouched.
+    """
+    to_remove = []
+    for sock in srv.sockets:
+        try:
+            if sock.getsockopt(socket.SOL_SOCKET, socket.SO_ACCEPTCONN):
+                loop.remove_reader(sock.fileno())
```

Comment:
Good point. It's not supported by Windows IOCP.
BaseEventLoop has `_stop_serving(sock)` method but I don't want to use private API.